common/app/common/Edition.scala

```diff
@@ -82,8 +81,8 @@ object Edition {
   }
 
   def apply(request: RequestHeader): Edition = {
-    val cookieValue = editionFromRequest(request)
```

Comment:
Renaming because `cookieValue` is misleading. This [method](https://github.com/guardian/frontend/blob/6cc188128f816e5b746018a82cabe16de889abcf/common/app/common/Edition.scala#L67-L82) returns the edition from a parameter, header or a cookie.